dom/src/mockDOMSource.ts

```diff
@@ -76,14 +90,13 @@ export class MockedDOMSource implements DOMSource {
         return vnode;
       } else {
         vnode.sel += `.${SCOPE_PREFIX}${scope}`;
+
         return vnode;
       }
     });
   }
 }
 
-export function mockDOMSource(
-    streamAdapter: StreamAdapter,
-    mockConfig: Object): DOMSource {
-  return new MockedDOMSource(streamAdapter, mockConfig);
+export function mockDOMSource(options: MockedDOMSourceOptions): DOMSource {
+  return new MockedDOMSource(options);
 }
```

Comment:
It's enough to just adapt the inputs here instead of just passing them directly.